aggregators/1.0/modules/ROOT/pages/aggregators-module-reference.adoc

```diff
@@ -1,46 +1,45 @@
-= Aggregators Module Reference - Mule 4
+= Aggregators Module Reference 1.0 - Mule 4
 :page-aliases: connectors::aggregator/aggregators-module-reference.adoc
 
 Support Category: https://www.mulesoft.com/legal/versioning-back-support-policy#anypoint-connectors[Select]
 
-Aggregators Module v1.0
-
 Release Notes: xref:release-notes::connector/aggregators-module-release-notes.adoc[Aggregators Module Release Notes]
 
-An Aggregator is a component that receives some data, processes it to extract some value and then adds that value to a list of aggregated elements. After that, and depending on the configuration, the list of elements is sent to a set of components.
+An aggregator scope is a component that receives some data, processes the data to extract some value, and then adds that value to a list of aggregated elements. After that process is complete, and depending on the aggregator scope configuration, a set of components in the flow process the list of elements.
 
-Aggregators are pass-through routers, meaning that the same data that is received by the aggregator is going to be processed by the components that follow it. The only modifications are the propagation of variables in case any is set while executing any of the aggregation routes.
+Aggregators are pass-through routers. The same data that an aggregator scope receives is processed by the components that follow the aggregator scope. 
+The only modifications are the propagation of variables in case any are set while executing any of the aggregation routes.
 
-When an aggregator releases the stored values, the list of aggregated elements is processed either by a route within the aggregator itself or in another Mule Flow, through an aggregator listener. All of that depends on the type and configuration of the aggregator being used.
+When an aggregator scope releases the stored values, a route component (*Incremental aggregation* or *Aggregation complete*) processes the list of aggregated elements within the aggregator scope itself or in another Mule flow, through an *Aggregator listener* source. The process depends on the configuration of the aggregator scope used.
 
 == Configurations
 
-There is no configuration for Aggregators because the module is based only on operations. Each different type of aggregator is a different operation that configurable through its parameters.
-
-However, there are a couple of common parameters that are worth mentioning beforehand:
+There is no global configuration for the Aggregator Module because the module uses scopes. Each type of aggregator is a different scope that is configurable through its parameters. The following parameters are common in each scope:
```

Comment:
```suggestion
There is no global configuration for the Aggregator Module because the module uses scopes. Each type of aggregator is a different scope that is configurable through its parameters. The following parameters are common in each scope:
```